CHANGELOG.md

```diff
@@ -0,0 +1,413 @@
+# Changelog
+
+## v1.7.5
+
+- ✨add an option to silent multiple Init errors (#310)
+
+## v1.7.4
+
+- 🐛 replace console.error by console.warn when cookies are not supported (#307)
+- :lock: upgrade vulnerable packages (#306)
+
+## v1.7.3
+
+- :bug:[RUMF-403] fix checkURLSupported (#302)
+- :white_check_mark: add cbt error case (#299)
```

Comment:
it could be nice to automatically convert emoji name to emoji icon, something like https://www.npmjs.com/package/emoji-name-map could help

Reply:
It's weird. Some of the commit have the unicode value and others have the `:name:` representation. I wonder what's doing the conversion? 🤔

Reply:
Done 😄

Reply:
IMO, it is done by us manually.
On my side, I some time reuse the converted message from github as the PR desciption to avoid :xxx: in the PR name.

Reply:
It could also be added as a pre-commit hook at some point